src/app/components/MainMenu.js

```diff
@@ -0,0 +1,77 @@
+import createVisualImage from './VisualImage';
+import createLogo from './Logo';
+import createButtonRed from './ButtonRed';
+import createWhiteButtonWithIcon from './ButtonWhiteWithIcon';
+import createGameModeName from './GameModeName';
+import ModeMenu, { menuOption } from './ModeMenu';
+import createModeRules from './ModeRules';
+
+// Creating page content
+function PageContent(optionMode = { selectOption: 'People' }) {
+  const section = document.querySelector('.section');
+  section.textContent = '';
+  const gameWrapper = document.createElement('div');
+  const buttonsWrapper = document.createElement('div');
+  const visualImage = createVisualImage(
+    '../../../static/assets/img/modes/people/1.jpg',
+  );
+  const modeRules = createModeRules(optionMode.selectOption);
+  // Insert score table for player #18;
+  const buttonPlay = createButtonRed('play the game');
+  const buttonRulesRanking = createWhiteButtonWithIcon(
+    'Hall of fame',
+    'fa',
+    'fa-id-badge',
+  );
+  const gameModeInfo = createGameModeName(
+    `Who is this ${optionMode.selectOption}?`,
+  );
+  gameWrapper.classList.add('section__wrapper');
+  buttonsWrapper.classList.add('section__wrapper__buttons');
+  section.append(visualImage, gameWrapper);
+  gameWrapper.append(gameModeInfo, modeRules, buttonsWrapper);
+  buttonsWrapper.append(buttonRulesRanking, buttonPlay);
+
+  // Changing the rules and ranking view by pressing the button
+  let rules = false;
+  function handleChangeOfView() {
+    if (!rules) {
+      modeRules.remove();
```

Comment:
remember about the cleanup: 🧹 
https://stackoverflow.com/questions/36998877/what-is-the-difference-between-remove-and-removechild-method-in-javascript 👀

Reply:
Doesn't GC do that?